WordPress/Classes/ViewRelated/Gutenberg/Processors/GutenbergMediaFilesUploadProcessor.swift

```diff
@@ -30,7 +30,7 @@ class GutenbergMediaFilesUploadProcessor: Processor {
                 return mediaFile
             }
 
-            guard let newURL = StoryPoster.filePath?.appendingPathComponent("\(self.serverMediaID)") else {
+            guard let newURL = StoryPoster.filePath?.appendingPathComponent(self.serverMediaID.description) else {
```

Comment:
Maybe `String(self.serverMediaID)` rather than a call to `description`.  I'm not sure if calling `description` would be a poor practice, but the explicit cast seems more straightforward, and we're not privy to how `description` is implemented under the hood.